src/features/mediaSelector/listMedia.tsx

```diff
@@ -18,8 +19,14 @@ export const MediaList: FC<MediaSelectorMediaListProps> = ({
 
   const isVideo = (mediaUrl: string | undefined) => {
```

Comment:
Make it reusable

Reply:
And replace where we have it

Reply:
created 2 new files inside features folder / utility : 
1.filterContentType.ts contains isVideo function.

2.filterExtensions.ts: contains listing  of possible extensions